hoot-test/src/main/cpp/hoot/test/main.cpp

```diff
@@ -545,7 +590,10 @@ int largestOptionNameSize()
 
 void usage(char* argv)
 {
-  cout << argv << " Usage:" << endl;
+  QString commandName(argv);
```

Comment:
Just cleaned it up to only show `HootTest` instead of the full path to the executable.